scripts/sandbox.js

```diff
@@ -90,22 +122,22 @@ function setStarRating() {
         let star = ratingsList[i];
         star.classList.remove("active");
       } else if (ratValue == 1) {
-        if (i == 1 || i == 2 || i == 3 || i == 4){
+        if (i == 1 || i == 2 || i == 3 || i == 4) {
```

Comment:
All of these little changes in your diff tell me your team have not agreed on a formatting beforehand, and not enabled consistent auto-formatting across the team. That's always a good idea, to keep irrelevant changes out of pull requests, and avoid merge conflicts.